src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -269,6 +269,7 @@
 	<bean id="preferencesDao" parent="preferencesDao-parentBean"/>
 	<bean id="preferencesDao-parentBean" abstract="true" class="${kfs.documentstore.class}">
 		<property name="documentStoreClient" ref="documentStoreClient" />
+		<property name="tableNamePrefix" value="${kfs.documentstore.tablename.prefix}" />
```

Comment:
I don't think we would ever actual user our custom Mongo implementation - if we decide to go back to Mongo at some point in the future, I think we would eliminate the entire customization. That said, it probably makes sense to make a change for consistency. I will add default get/set implementations in a new interface as that seems like the most straightforward thing to do at this point and it also lets me try out Java8 default interface methods.